src/scripts/main.js

```diff
@@ -2,4 +2,33 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+  const wallRectangle = wall.getBoundingClientRect();
+
+  let x = e.clientX - wallRectangle.left - spider.width + spider.width / 2;
+
+  let y = e.clientY - wallRectangle.top - spider.height + spider.height / 2;
```

Comment:
The calculation for `y` should be `e.clientY - wallRectangle.top - spider.height / 2` to correctly center the spider on the click position.